bin/morgue.js

```diff
@@ -5957,41 +5959,54 @@ function uint128ToUuid(uint128) {
   return parts.join("-");
 }
 
+function uint128ToIpv6(uint128) {
+  // already an ipv6 address
+  if (typeof uint128 === 'string' && uint128.includes(':')) {
+    return uint128;
+  }
+
+  const bytes = Buffer.from(uint128, 'hex')
+  const parts = []
+
+  for (let i=0; i<16; i+=2) {
+    parts.push(bytes.subarray(i, i+2).toString('hex').padStart(1, '0'))
```

Comment:
It's customary for IPv6 address strings to omit nibbles that are zero.  Do you want to try to account for that?  There might be a library to handle this for you.

Reply:
I updated the code with `ip6addr` library.